org.grisbi.Grisbi.yml

```diff
@@ -0,0 +1,62 @@
+app-id: org.grisbi.Grisbi
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.gnome.Sdk.Locale
+command: grisbi
+
+rename-desktop-file: grisbi.desktop
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
```

Comment:
it if supports wayland then this shuld be `fallback-x11`.

Also `--share=ipc` is necessary regardless.